VBAProjectFiles/RunOpenStudioCLI.bas

```diff
@@ -79,65 +79,12 @@ Sub RunOpenStudioCLI()
             End If
             Range("Status").Offset(5, 1) = "beendet (" & WorksheetFunction.Round((Time - Startzeit_indv) * 86400, 1) & " s)"
             Startzeit_indv = Time
-                
+
             DoEvents
         Else
             Range("Status").Offset(4, 0) = "Fehler bei generieren der CSV Datei"
-            
+
             DoEvents
         End If
     End If
 End Sub
-
-'Sub RunOpenStudioWithProgressMessagesInExcel(Argument As String)
```

Comment:
I don't quite see where this functionality moved to or did we not really use this part of the code?